src/main/java/com/umbr3114/models/PostModel.java

```diff
@@ -16,6 +20,7 @@
     public String dropId;
     public String idString;
     //public List<CommentModel> comments;
+    public int startAfter;
```

Comment:
startAfter should not be in the PostModel. Posts do not have to contain data about their position in a list. Also, id's aren't stored into `int` variables.